DH_Engine/Classes/DHAxisMarksmanRoles.uc

```diff
@@ -0,0 +1,20 @@
+//==============================================================================
+// Darkest Hour: Europe '44-'45
+// Darklight Games (c) 2008-2023
+//==============================================================================
+
+class DHAxisMarksmanRoles extends DHAxisRoles
+    abstract;
+
+defaultproperties
+{
+    //historically, there was no "marksman" type role within the wehrmacht or Waffen-SS, however the ZF41 was given out to fullfill this role and there was no distinction between snipers with or without 
+    MyName="Marksman"   //https://www.youtube.com/watch?v=sBstpqUAniw <--- reference to marksman being a thing in the wehr and SS
+    AltName="Scharfsch�tze" //the native role name reflects this, but the translated name is there so that people can understand that this isnt a true "sniper"
```

Comment:
Use Windows -1252 encoding when saving this file, otherwise the text gets corrupted